audit-log/README.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: pattern
+title: Audit Log
+folder: audit-log
+permalink: /patterns/audit-log/ 
+categories:
+- creational 
+language: en 
+tags:
+- Security
+- Data Access
```

Comment:
I did not notice this in my initial review, but, I don't think the data access tag is appropriate for this pattern. What do you think?

Reply:
Im not so sure, I may need to rely on iluwatar's opinion on this one. I think that it is security because it writes an audit of everything important that happens, however, unlike the other temporal patterns, this isnt an interface for reading data. 

I think I will leave it as is for now, because it does create a new form for data to be accessed, that is, through reading and 'reconstructing' values from the log itself.

Reply:
Yeah that is fair enough. The tags are somewhat ambiguous I think. 

I agree it is probably best to leave the tags are they are and get iluwatar's opinion.